cmd/cspc-operator/app/operations.go

```diff
@@ -256,3 +379,27 @@ func (pc *PoolConfig) ClaimBD(bdName string) error {
 	}
 	return nil
 }
+
+func isPoolSpecBlockDevicesGotReplaced(
```

Comment:
add comment here as well , about the bd comparsion

Reply:
updated with a code comments